.devops/deploy-pipelines.yml

```diff
@@ -160,6 +160,12 @@ stages:
               TEST_FILE_PREFIX: "DEV-wallet-integration-test-paypal"
               API_TEST_FILE: "api-tests/wallet_payment_api.paypal.tests.json"
               WALLET_TOKEN_TEST: $(WALLET_TOKEN_TEST_DEV)
+          - template: azure-templates/api-tests.yml
+            parameters:
+              ENV_FILE: "api-tests/wallet_local_migration.environment.json"
+              API_TEST_FILE: "api-tests/wallet_local_migration.collection.tests.json"
```

Comment:
I don't think you can use this collection to run tests after the deploy. 
We are using as HOSTNAME localhost. What do you think ?

Reply:
yes you are right, I realize now that there is [pagopa-wallet-tests](https://github.com/pagopa/pagopa-wallet-tests/tree/main/api-tests). Should I create an environment file here for post deployment testing?

Reply:
Yes you can open a PR on that repository by adding the collection.
Let's see if we can reuse one of env files already present otherwise a new one is fine too 🙂

Reply:
This new api requires subscription key, so you would need a new environment variable containing it. Since it is a secret, do you use a secret file? Wallet_token is retrieved from pipeline environment variable, but adding a new ones require a new PR on pipeline definition, is right?

Reply:
The key you need is the one contained in the $(WALLET_TOKEN_TEST_DEV) variable configured in the code review pipeline on the azure devops repository and retrieved from the key vault.

It will be up to the api-tests template to replace the value when you launch tests with newman https://github.com/pagopa/pagopa-wallet-service/blob/1bb23266f3903d4779c20d3ee8eeeb6177bbf4fd/.devops/azure-templates/api-tests.yml#L16

Make sure you have called in your env value the token variable like this : WALLET_TOKEN_TEST

Reply:
Wa have create a dedicated token called `MIGRATION_WALLET_TOKEN_TEST_DEV` on pipeline env, and then used to run tests as you can see:
```yaml
        - template: azure-templates/api-tests.yml
          parameters:
            TEST_DISPLAY_NAME: "Wallet migration tests"
            ENV_FILE: "api-tests/dev.migration.envs.json"
            API_TEST_FILE: "api-tests/wallet_migration_api.tests.json"
            TEST_FILE_PREFIX: "DEV-wallet-integration-test-migration"
            WALLET_TOKEN_TEST: $(MIGRATION_WALLET_TOKEN_TEST_DEV)
            MIGRATION_API_KEY_TEST: $(MIGRATION_API_KEY_TEST_DEV)
```